src/main/resources/openapi/annotations.yaml

```diff
@@ -47,6 +47,25 @@ paths:
             application/json+problem:
               schema:
                 $ref: './problem.yaml'
+  /annotations/{annotation_id}:
+    delete:
+      tags:
+        - Annotations
+      summary: Deletes an annotation.
+      operationId: deleteAnnotation
+      parameters:
+        - in: path
+          name: annotation_id
+          schema:
+            type: integer
+          required: true
+      responses:
+        '204':
+          description: Annotation has been deleted
+        '403':
```

Comment:
should we be including references to the problem.yaml here?

Reply:
possibly, will take a look